### PR TITLE
automated deployment with parallel execution

### PR DESCRIPTION
**Pull request by xAt0mZ**
Modified the scripts to be generic. Getting nodes number from env, based on env variable name defined in Vagrantfile.

Also has parallel execution to speed up the kernel install / application and CIO install (tasks runs in parallel on all nodes of the cluster).

Still need to run 2 scripts and to wait cluster to be completely restarted before running CIO install + cluster setup.

### CODE REVIEW

File: 01-downgrade-kernel.sh [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
+    do
+        (vagrant ssh storidge-$i --no-tty -c "${INSTALL_KERNEL}" | sed "s/^/[storidge-$i] /") &
+    done
+    wait
+
+    echo " ===== KERNEL INSTALLED ON ALL MACHINES ====="
+}
+
+# Rollback kernel on all machines
+rollback() {
```
Comment (deviantony) [inline 01-downgrade-kernel.sh]: Not a really a rollback but simply a downgrade, should rename.

File: README.md [hunk 2]
```diff
@@ -57,8 +57,7 @@ After the setup is completed, you can access the Portainer interface on *10.0.9.
 Use the following scripts to setup the cluster:
 
 ```
-./01-downgrade-kernel.sh
+./1-downgrade-kernel.sh
 # Wait for cluster to reboot entirely
-./02-install-cio.sh
-./03-setup-cluster.sh
+./2-install-cio-and-setup-clister.sh
```
Comment (deviantony) [inline README.md]: Should'nt it be `./2-install-cio-and-setup-cluster.sh all` ?

**Review by deviantony — CHANGES_REQUESTED** [review]
Dope ! Few small things to change regarding naming.

File: README.md (hunk 2 above)
Comment (xAt0mZ) [reply README.md]: Forgot to update the README, will do !

**Review by deviantony — APPROVED** [review]
LGTM

**Pull request merged**